geomesa-fs/geomesa-fs-datastore/src/main/scala/org/locationtech/geomesa/fs/storage/converter/ConverterStorage.scala

```diff
@@ -0,0 +1,121 @@
+/***********************************************************************
+ * Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.fs.storage.converter
+
+import java.io.Serializable
+import java.net.URI
+import java.util
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.geotools.data.Query
+import org.locationtech.geomesa.convert.{ConfArgs, ConverterConfigResolver, SimpleFeatureConverter, SimpleFeatureConverters}
+import org.locationtech.geomesa.fs.FileSystemDataStoreParams
+import org.locationtech.geomesa.fs.storage.api._
+import org.locationtech.geomesa.fs.storage.common.{StoragePartition, PartitionScheme}
+import org.locationtech.geomesa.utils.geotools.{SftArgResolver, SftArgs}
+import org.opengis.feature.simple.SimpleFeatureType
+
+
+class ConverterStorageFactory extends FileSystemStorageFactory {
+  override def canProcess(params: util.Map[String, Serializable]): Boolean = {
+    params.containsKey("fs.path") &&
+    params.containsKey("fs.encoding") &&
+      params.get("fs.encoding").asInstanceOf[String].equals("converter")
```

Comment:
already done as not this pr. but i have a ticket: https://geomesa.atlassian.net/browse/GEOMESA-1932